netsim/modules/lag.yml

```diff
@@ -26,6 +26,6 @@ attributes:
 
   # Copy only these L2 attributes into LAG physical link members
   lag_l2_ifattr:
-    lag.ifindex:
     mtu:
     bandwidth:
+    stp:
```

Comment:
This is wrong. STP is not running on individual member links (otherwise one of them would be blocking). STP parameters SHOULD NOT be copied into LAG member links.

Reply:
removed